### PR TITLE
Octopus config for Trident

### PR DESCRIPTION
**Pull request by cbc02009**
This config matches the wiring diagram for the v2 in the voron docs, which will allow us to re-use it and save some work.

### CODE REVIEW

**Comment by cbc02009** [comment]
Still TODO:

- Remove references to depricated thermistors
- add leadscrew comments from Yeri's config.

**Comment by cbc02009** [comment]
Need to replace QGL with tilt

**Comment by telxoid** [comment]
Hiya--here's what I found.  Let me know if there are issues or the line numbers got wacky.

Line 142/143:  Dupes.
Line: 145, Z homing speed.  35 is way fast.
`homing_speed: 8.0 # Leadscrews are slower than 2.4, 10 is a recommended max.`

Lines 153, 154, 172, 173, 191, 192: All Z currents are high.  Use 0.6 to start.

Line 235:  I typically leave PA commented for initial config.

Line 362-364:  I'd leave this commented so someone *has* to config it.

Line 366-414: Remove QGL.

Added z_tilt section as below in Macros section:

```
[z_tilt]
##	Use Z_TILT_ADJUST to level the bed .
##	z_positions: Location of toolhead

##--------------------------------------------------------------------
## Uncomment below for 250mm build
z_positions:
	-50, 18
	125, 298
	300, 18
points:
	30, 5
	125, 195
	220, 5

## Uncomment below for 300mm build
#z_positions:
#	-50, 18
#	150, 348
#	350, 18
#points:
#	30, 5
#	150, 245
#	270, 5

## Uncomment below for 350mm build
#z_positions:
#	-50, 18
#	175, 398
#	400, 18
#points:
#	30, 5
#	175, 295
#	320, 5


##--------------------------------------------------------------------

speed: 200
horizontal_move_z: 10
retries: 5
retry_tolerance: 0.0075
```
Config doesn't have any of the required stuff for Mainsail.  Maybe link to https://docs.mainsail.xyz/necessary-configuration at the top?

**Comment by cbc02009** [comment]
@telxoid Thanks for the great feedback! You don't want the safe_z_home section commented out because you need the printer to run to configure the A/B steppers before homing z. The negative values prevent the user from accidentally crashing into the bed by being over-eager and not following the startup guide.

In the past we've left references to mainsail/fluidd out of the stock config files, but it can't hurt to add commented lines for users to uncomment.  Let me see what the devs prefer.

File: Firmware/Octopus/Trident_Octopus_Config.cfg [hunk 1]
```diff
@@ -0,0 +1,533 @@
...
+speed:100
+z_hop:10
+
+[z_tilt]
+##  Use Z_TILT_ADJUST to level the bed .
+##  z_positions: Location of toolhead
+
+##--------------------------------------------------------------------
+## Uncomment below for 250mm build
+z_positions:
```
Comment (KiloQubit) [inline Firmware/Octopus/Trident_Octopus_Config.cfg:371]: I would comment the 250 section out too in the stock config.

Comment (cbc02009) [reply Firmware/Octopus/Trident_Octopus_Config.cfg:371]: Good catch. Will fix.

**Comment by cbc02009** [comment]
@KiloQubit any other concerns with this config? I've had quite a few users test it out for me in the trident channel.
Would you rather I move this to usermods instead?

**Review by KiloQubit — APPROVED** [review]

**Pull request merged**